docs/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "material-ui-docs",
```

Comment:
What happens if we just name this `docs`?

I want to avoid confusion with `@material-ui/docs` and prepare for yarn berry where naming this `docs` would naturally enable `yarn docs:typescript`.

Reply:
> What happens if we just name this docs?

Works perfectly fine.

> yarn berry where naming this docs would naturally enable yarn docs:typescript

Sweet :+1: